net/pfSense-pkg-frr/files/usr/local/pkg/frr/inc/frr_ospf.inc

```diff
@@ -169,6 +169,9 @@ function frr_generate_config_ospf_areas() {
 
 function frr_generate_config_ospf_interfaces() {
 	global $config, $passive_interfaces, $frr_integrated_config;
+	if (is_array($config['installedpackages']['frrospfd']['config'])) {
+		$ospfd_conf = &$config['installedpackages']['frrospfd']['config'][0];
+	}
```

Comment:
It's better to initialize this config item with `init_config_arr()` as done few lines below

Reply:
done